docs/guide/usage.md

```diff
@@ -0,0 +1,86 @@
+## Create a Store with Rodux
+Create your store as normal with [Rodux](https://github.com/Roblox/Rodux):
+
+```lua
+local store = Rodux.Store.new(function(state, action)
+	state = state or {
+		value = 0,
+	}
+
+	if action.type == "increment" then
+		return {
+			value = state.value + 1,
+		}
+	end
+
+	return state
+end)
+```
+
+## Connect with `connect`
+Use `RoactRodux.connect` to retrieve values from the store and use them in your [Roact](https://github.com/Roblox/Roact) component:
+
+```lua
+-- Write your component as if Rodux is not involved first.
+-- This helps guide you to create a more focused interface.
+
+local function MyComponent(props)
+	-- Values from Rodux can be accessed just like regular props
+	local value = props.value
+	local onClick = props.onClick
+
+	return Roact.createElement("ScreenGui", nil, {
+		Label = Roact.createElement("TextButton", {
+			-- ...and used in your components!
+			Text = "Current value: " .. value,
+			Size = UDim2.new(1, 0, 1, 0),
+
+			[Roact.Event.Activated] = onClick,
+		})
+	})
+end
+
+-- `connect` accepts two optional functions:
+-- `mapStateToProps` accepts your store's state and returns props
+-- `mapDispatchToProps` accepts a dispatch function and returns props
+
+-- Both functions should return a table containing props that will be passed to
+-- your component!
+
+-- `connect` returns a function, so we call that function, passing in our
+-- component, getting back a new component!
+MyComponent = RoactRodux.connect(
+	function(state)
+		-- mapStateToProps is run every time the store's state updates.
+		-- It's also run whenever the component receives new props.
+		return {
+			value = state.value,
+		}
+	end,
+	function(dispatch)
+		-- mapDispatchToProps only runs once, so create functions here!
+		return {
+			onClick = function()
+				dispatch({
+					type = "increment",
+				})
+			end,
+		}
+	end
+)(MyComponent)
+```
+
+## Add a `StoreProvider`
+Finally, when you render your Roact application, wrap the top-level component in a `RoactRodux.StoreProvider`:
```

Comment:
I think it might be a little clearer to put this section above the `connect` section.  It makes sense to me to create the store, provide it to the tree, and then connect your components to it.  Obviously the sequence in which the steps are taken isn't really important, but to me its easiest to reason with it that way.